invenio_circulation/config.py

```diff
@@ -97,8 +102,9 @@
                                  ':json_v1_search'),
         },
         list_route='/circulation/loan/',
-        item_route='/circulation/loan/<pid(loan_pid):pid_value>',
+        item_route='/circulation/loan/<{0}:pid_value>'.format(_Loan_PID),
         default_media_type='application/json',
+        links_factory_imp=_CIRCULATION_ACTION_LINKS_FACTORY,
```

Comment:
Why do you have to change the `links_factory_imp` for the default REST endpoints?

Reply:
because when you create a loan it's gonna return back in the links section the next available actions(request, checkout) in order for the ui to decide. In general links will give you back the available actions based on the loans current state.